src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidator.java

```diff
@@ -34,21 +34,20 @@ public TrustCorporateValidator(AddressDtoValidator addressDtoValidator) {
         this.addressDtoValidator = addressDtoValidator;
     }
 
-    public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext) {
+    public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext, boolean isForUpdateOrRemove) {
         for (TrustDataDto trustDataDto : trustDataDtoList) {
             List<TrustCorporateDto> corporates = trustDataDto.getCorporates();
-
             if (!CollectionUtils.isEmpty(corporates)) {
                 for (TrustCorporateDto trustCorporateDto : corporates) {
-                    validateTrustCorporateDto(trustCorporateDto, errors, loggingContext);
+                    validateTrustCorporateDto(trustCorporateDto, trustDataDto.getCreationDate(), errors, loggingContext, isForUpdateOrRemove);
                 }
             }
         }
         return errors;
     }
 
-    private void validateTrustCorporateDto(TrustCorporateDto trustCorporateDto, Errors errors,
-            String loggingContext) {
+    private void validateTrustCorporateDto(TrustCorporateDto trustCorporateDto, LocalDate trustCreationDate, Errors errors,
```

Comment:
Have you tested an Update journey, with changes but NOT going into the Corporate trustee to edit it, but just continuing?

I would expect an error to arise from line 73 in this class. I had to make a change for the Individual Trustee. See https://github.com/companieshouse/overseas-entities-api/pull/400/commits/1a0154e6e1a3f7ee58071abd8e126fe3b55bb9bf.

Reply:
Have tried this journey and seen no errors.

Reply:
Not sure line 73 is related to this PR should this be fixed here?

Reply:
Ok. Tested this with @mattch1 and seems that the issue present for Individual trustees is not there for corporates. No action therefore required.